cumulus_library/databases.py

```diff
@@ -41,6 +41,51 @@ def fetchall(self) -> Optional[list[list]]:
         pass
 
 
+class DatabaseParser(abc.ABC):
+    """Parses information_schema results from a database"""
+
+    def _parse_found_schema(self, expected: dict[dict[list]], schema: list[tuple]):
+        """Checks for presence of field for each column in a table
+
+        This naieve method is a first pass, ignoring complexities of differing
+        database variable types, just iterating through looking for column names.
+
+        TODO: on a per database instance, consider a more nuanced approach
+        if needed
```

Comment:
We both suspect it will be at some point yeah? I'm not hopeful we can band-aid this too much, but I think this sets us on a good path.